data visualisations/dynamicVisu.html

```diff
@@ -0,0 +1,62 @@
+<html>
```

Comment:
Well done on the visualisation! However it is not enough to just put the html file like that on the website... You need to embedded as a page. Maybe see with me or @Bayar4B he seems to have a pretty good understanding of how jekyll builds pages. If you can embed this in the current website it will be perfect

Reply:
Either embed the javascript code in a markdown file and include only the div for the animation or add a markdown header description to your html like we have in the layout pages that @Bayar4B modified